README.md

```diff
@@ -46,7 +46,7 @@ Implement `prepareData` higher order function, which takes an object of filter p
 
 **Observations:**
 
-- Missions should appear in inverse chronological order (sort), with the exception of those that carried more payloads should appear first.
+- Missions should appear in inverse chronological order (sorted by the `launch_date_utc` field), with the exception of those that carried more payloads should appear first.
```

Comment:
Not relevant to this PR, but should it be "carry" instead of "carried"? When I reread this with fresh context, Present Simple feels like a better fit here 😅 Although it might've been a long day.
```suggestion
- Missions should appear in inverse chronological order (sorted by the `launch_date_utc` field), with the exception of those that carried more payloads should appear first.
```

Also, here's GPT take on that: `with the exception that those carrying more payloads should appear first.`

<img width="729" alt="Screenshot 2023-11-28 at 17 40 43" src="https://github.com/adjust/fe-hometask/assets/25829136/427e71ad-c946-4a10-9f14-bafdd8a905cc">

Reply:
Updated, thank you!